.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.10.0
+current_version = 2.11.1
```

Comment:
I also took this opportunity to merge `master` back to `develop`.

Reply:
Thank you.